bedrock/settings/static_media.py

```diff
@@ -1088,6 +1088,14 @@
         ),
         'output_filename': 'js/common-bundle.js',
     },
+    'common-ie8': {
+        'source_filenames': (
+            'js/libs/jquery-1.11.3.min.js',
+            'js/ie8/mozilla-utils.js',
```

Comment:
One thing maybe missing here is GA. Do we need to make sure both `js/base/core-datalayer.js`, and `core-datalayer-init.js` are IE8 compatible?